tests/ops/api/v1/endpoints/test_dataset_endpoints.py

```diff
@@ -288,7 +288,7 @@ def test_put_validate_dataset_invalid_category(
         )
         assert response.status_code == 422
         details = json.loads(response.text)["detail"]
-        assert ["body", "collections", 0, "fields", 0, "data_categories"] in [
+        assert ["collections", 0, "fields", 0, "data_categories"] in [
```

Comment:
Updated validation was added to check data_categories against those in the database, not those in the static default data categories list.  To do this, I couldn't use a regular pydantic validator, but needed to do this after the fact, when I could access a database session.  So the response has adjusted as the request body itself isn't being validated here.